DESCRIPTION

```diff
@@ -33,11 +33,13 @@ Depends:
     R (>= 4.0)
 Imports: 
     cli,
+    desc,
     grDevices,
     grid,
     gtable (>= 0.1.1),
     isoband,
     lifecycle (> 1.0.1),
+    pkgload,
```

Comment:
I don't like having to take on these dependencies.
Would it be bad to have a homebrew solution to parsing the description file?